config/routes.rb

```diff
@@ -116,6 +116,10 @@
     member do
       get :schedule
     end
+
+    member do
+      get :allevents
```

Comment:
I am not sure of what you mean with `conference/<slug>/events`.

In some places the all events is called program. Maybe program is a better name.